components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -220,8 +220,7 @@ private void init(String projectName, String configPath, String deploymentConfig
                 copyFile.setIsBallerinaConf("true");
                 copyFile.setSource(CmdUtils.getResourceFolderLocation() + File.separator + CliConstants.GW_DIST_CONF
                         + File.separator + CliConstants.MICRO_GW_CONF_FILE);
-                copyFile.setTarget(File.separator + CliConstants.WSO2 + File.separator + CliConstants.MGW
```

Comment:
Shall we move this docker related logic to separate method

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/966/commits/fe1f68c2c9d1d39e961d2664256249498d69c5aa